test/jdk/javax/swing/JColorChooser/Test4319113.java

```diff
@@ -50,56 +38,65 @@
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
 
-public class Test4319113
-extends JApplet
-implements ActionListener {
-    private final JFrame frame = new JFrame("frame");
-    private JComboBox cbPlaf;
-
-    @Override
-    public void init() {
-        try {
-            java.awt.EventQueue.invokeLater( () -> {
-                Test4319113.this.frame.setLayout(new GridLayout(2, 1));
-                Test4319113.this.show(Test4319113.this.frame);
-            });
-        }catch(Exception ex) {
-            ex.printStackTrace();
-        }
+/*
+ * @test
+ * @bug 4319113
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
+ * @summary Tests the open JColorChooser behavior on LaF change.
+ * @run main Test4319113
```

Comment:
This needs to specify the test is manual.

Reply:
i've separated the two tests into two separate PRs